operator/gitops/argocd/pipeline-service/pipelines-as-code/bugfix.yaml

```diff
@@ -0,0 +1,79 @@
+# Solves https://issues.redhat.com/browse/RHTAPBUGS-256 until we deploy
+# PaC through OpenShift Pipelines 1.10 or greater.
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: pac-secret-manager
+  namespace: pipelines-as-code
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: pac-secret-manager
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
+    verbs:
+      - list
+      - delete
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: pac-secret-manager
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: pac-secret-manager
+subjects:
+  - kind: ServiceAccount
+    name: pac-secret-manager
+    namespace: pipelines-as-code
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: pac-secret-reaper
+  namespace: pipelines-as-code
+spec:
+  concurrencyPolicy: Forbid
+  schedule: "*/10 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - image: registry.redhat.io/openshift4/ose-cli:v4.12@sha256:9f0cdc00b1b1a3c17411e50653253b9f6bb5329ea4fb82ad983790a6dbf2d9ad
+              command:
+                - /bin/bash
+                - -c
+                - |
+                  set -o errexit
+                  set -o nounset
+                  set -o pipefail
+
+                  dnf install -y jq
+
+                  kubectl get secrets --all-namespaces -o json | \
```

Comment:
Would it be better if I looped over namespaces instead of a single query over all namespaces?

I'd prefer not having a SOP, as it means that the user is already impacted.